api/queries/thread/reactions.js

```diff
@@ -8,6 +8,14 @@ export default async (
   _: any,
   { user, loaders }: GraphQLContext
 ) => ({
-  count: reactionCount,
+  count:
+    typeof reactionCount === 'number'
+      ? reactionCount
+      : await loaders.threadReaction
```

Comment:
Why would we do this? Why not just return 0 if the reactionCount field on the db record isn't a number? Or - if it's not a number: why? Did we have a bug in the migration script? Do we have bad db data we need to clean up?

Overall this feels a bit hacky and would be nice to have the reactionCount field be either accurate or return 0 :)

Reply:
Because in this scenario the worst-case means we have bad data in the db and we're addressing a symptom not the underlying cause. Seems like we'll be a lot happier if we can know that the db data is correct.